### PR TITLE
[#159706347] Updating TextInput to support Placeholder & PlaceholderTextColor passed as props

### PR DESCRIPTION
**Pull request by armandgray**

### CODE REVIEW

**Review by ArunJ98 — APPROVED** [review]
👍

File: index.js [hunk 1]
```diff
@@ -374,6 +384,8 @@ class TagInput extends Component {
                   blurOnSubmit={false}
                   onKeyPress={this.onKeyPress}
                   value={text}
+                  placeholder={this.props.placeholder}
```
Comment (leoleblanc) [inline index.js:387]: what would happen here if no placeholder was given?  would it just be undefined and there'd be nothing?

Comment (armandgray) [reply index.js:387]: The same thing that happens now... no placeholder will show

Comment (leoleblanc) [reply index.js:387]: 👍

**Review by leoleblanc — APPROVED** [review]
🔥

**Pull request merged**